package.json

```diff
@@ -1,127 +1,129 @@
-    "repository": {
-        "type": "git",
-        "url": "git+https://github.com/citizensadvice/design-system-testing.git"
-    },
-    "author": "Citizens Advice",
-    "license": "UNLICENCED",
-    "bugs": {
-        "url": "https://github.com/citizensadvice/design-system-testing/issues"
-    },
-    "homepage": "https://github.com/citizensadvice/design-system-testing#readme",
-    "devDependencies": {
-        "@babel/cli": "^7.10.5",
-        "@babel/core": "^7.11.4",
-        "@babel/plugin-proposal-class-properties": "^7.10.4",
-        "@babel/plugin-transform-runtime": "^7.11.0",
-        "@babel/preset-env": "^7.11.0",
-        "@babel/preset-typescript": "^7.10.4",
-        "@storybook/addon-a11y": "^5.3.19",
-        "@storybook/addon-actions": "^5.3.19",
-        "@storybook/html": "^5.3.19",
-        "@storybook/preset-typescript": "^3.0.0",
-        "@types/jest": "^26.0.10",
-        "@types/jsdom": "^16.2.3",
-        "@typescript-eslint/eslint-plugin": "^3.10.1",
-        "@typescript-eslint/parser": "^3.10.1",
-        "autoprefixer": "^9.8.2",
-        "babel-eslint": "^10.1.0",
-        "babel-loader": "^8.1.0",
-        "chalk": "^4.1.0",
-        "cssnano": "^4.1.10",
-        "dotenv": "^8.2.0",
-        "eslint": "^7.7.0",
-        "eslint-config-airbnb-base": "^14.2.0",
-        "eslint-plugin-import": "^2.22.0",
-        "eslint-plugin-prettier": "^3.1.4",
-        "fork-ts-checker-webpack-plugin": "^5.1.0",
-        "fs-extra": "^9.0.1",
-        "git-state": "^4.1.0",
-        "http-server": "^0.12.3",
-        "husky": "^4.2.5",
-        "inquirer": "^7.3.3",
-        "jest": "^26.4.2",
-        "js-beautify": "^1.13.0",
-        "jsdom": "^16.4.0",
-        "lint-staged": "^10.2.13",
-        "markdown-loader": "^5.1.0",
-        "moment": "^2.27.0",
-        "node-sass": "^4.14.1",
-        "npm-run-all": "^4.1.5",
-        "postcss": "^7.0.32",
-        "postcss-url": "^8.0.0",
-        "prettier": "^2.1.1",
-        "raw-loader": "^4.0.1",
-        "resolve-url-loader": "^3.1.1",
-        "sass-loader": "^9.0.2",
-        "semver": "^7.3.2",
-        "shell-loader": "^1.2.1",
-        "simple-git": "^2.20.1",
-        "stylelint": "^13.6.1",
-        "stylelint-config-standard": "^20.0.0",
-        "stylelint-scss": "^3.18.0",
-        "stylelint-selector-bem-pattern": "^2.1.0",
-        "systemjs": "^6.5.0",
-        "typescript": "^3.9.6",
-        "webdriverio": "^6.4.2"
-    },
-    "dependencies": {},
-    "browserslist": [
-        "IE 11",
-        "last 2 Edge versions",
-        "last 2 Chrome versions",
-        "last 2 Firefox versions",
-        "last 2 Safari versions",
-        "last 2 ChromeAndroid versions",
-        "last 2 iOS versions"
-    ],
-    "lint-staged": {
-        "*.scss": [
-            "stylelint"
-        ]
-    },
-    "husky": {
-        "hooks": {
-            "pre-commit": "npm run pre-commit"
-        }
+  "name": "@citizensadvice/design-system",
+  "version": "1.8.3",
```

Comment:
This version needs to be 1.9.0 too.

Reply:
In general, the package version should not be changed manually. The `release` script will sort all that and update the CHANGELOG.